Shared/Extensions/ComboBox/ComboBoxSynchronizerBehavior.cs

```diff
@@ -1,16 +1,26 @@
-﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__
+﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS10_0_18362_0
 using System.Collections;
 using System.ComponentModel;
 using System;
 using System.Reflection;
 using Uno.Collections;
+#if WINUI
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Data;
+using SharedBinding = Microsoft.UI.Xaml.Data.Binding;
+#else
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Data;
 using SharedBinding = Windows.UI.Xaml.Data.Binding;
-#if __ANDROID__ || __IOS__ || __WASM__
+#endif
+#if !WINUI && (__ANDROID__ || __IOS__ || __WASM__)
 using static Windows.UI.Xaml.DependencyObjectStore;
 #endif
+#if WINUI && !WINDOWS10_0_18362_0
```

Comment:
@GabrielChouinardLetourneau did you add a task or comment to return to this?

Reply:
I received an answer from the Uno team: "`DependencyObjectStore` is Uno-specific API, so it is not available on Windows (which does not compile against Uno Platform but rather directly against Windows APIs. You will need to either avoid using it, or make the code that uses it run only on non-Windows targets"

Reply:
So the current way that Nventive.View of using the `AreDifferent` function that is from that API would be the correct way to go imo

Reply:
I added a !__WASM__ above the AreDifferent method since we do have access to `DependencyObjectStore` with the Netstandard target